domenetjenester/datavarehus/src/main/java/no/nav/foreldrepenger/datavarehus/v2/StønadsstatistikkUttakPeriodeMapper.java

```diff
@@ -317,5 +323,61 @@ private static boolean morFedrekvote(RelasjonsRolleType rolleType, Stønadskonto
         };
     }
 
+    private static List<StønadsstatistikkUttakPeriode> komprimerTidslinje(List<StønadsstatistikkUttakPeriode> perioder) {
+        var segmenter = perioder.stream()
+            .map(p -> new LocalDateSegment<>(p.fom(), p.tom(), new UttakSammenlign(p)))
+            .toList();
+        return new LocalDateTimeline<>(segmenter)
+            .compress(LocalDateInterval::abutsWorkdays, StønadsstatistikkUttakPeriodeMapper::likeNaboer, StønadsstatistikkUttakPeriodeMapper::slåSammen)
+            .stream()
+            .map(s -> s.getValue().tilUttakperiode(s.getFom(), s.getTom()))
+            .toList();
+    }
+
+    private record UttakSammenlign(PeriodeType type, StønadsstatistikkVedtak.StønadskontoType stønadskontoType,
```

Comment:
Hva er grunnen til at du lager en egen record for sammenligning? Kunne like greit sammenlignet StønadsstatistikkUttakPeriode ?